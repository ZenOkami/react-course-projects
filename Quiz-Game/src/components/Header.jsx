import React from "react";

const Header = () => (
    <header>
        <h1>Quiz Game</h1>
    </header>
);

export default Header;