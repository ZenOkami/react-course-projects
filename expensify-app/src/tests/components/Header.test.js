import React from "react";
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/headerPage'

test('Should Render Header Correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header />);

    expect(renderer.getRenderOutput()).toMatchSnapshot();

    console.log(renderer.getRenderOutput());
})