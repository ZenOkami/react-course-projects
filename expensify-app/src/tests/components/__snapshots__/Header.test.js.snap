// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Header Correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    className={[Function]}
    end={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    className={[Function]}
    end={true}
    to="/create"
  >
    Create an Expense
  </NavLink>
  <NavLink
    className={[Function]}
    end={true}
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
